### PR TITLE
Best practice

### PR DESCRIPTION
**Pull request by BonkeGcobo**
## Description

- [x] Follow Solid Principle in our codebase
- [x] Wrote DRY code

Happy Reviewing

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -42,19 +42,7 @@ def create_person
     age = gets.chomp
     print 'Name: '
     name = gets.chomp
```
Comment (Kingstalux) [inline app.rb]: - [ ] Good job so far 👍 but all requests for input and input validation should've been placed in a different file whose responsibilities are to request or validate input. Kindly correct the `create_person` method and make it more like the `list_people` or `create_book` methods as those have a single responsibility 👍 . You can read more on that [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid`).

File: app.rb [hunk 2]
```diff
@@ -86,8 +85,6 @@ def list_rentals
     puts 'Enter the person ID:'
     id = gets.chomp
     puts 'Rentals:'
```
Comment (Kingstalux) [inline app.rb]: - [ ] Same issue here as before. The `create_rentals` and `list_rentals` methods in the `app.rb` shouldn't ask a user for any input or validation but should simply either create the rental or list them. Kindly address this issue too 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo and @daudi13, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966


## Highlights 👏 
✅ App has the correct functionality 👍 
✅ Linters are passing 👏 
✅ Good commit history 🔥 



## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly add all partners of this project as authors in the README file to make it more professional 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @BonkeGcobo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**